siliconcompiler/tools/bambu/bambu.py

```diff
@@ -1,3 +1,4 @@
+import importlib
```

Comment:
why is this needed? (same for all tools) Can't you just import `.import`

Reply:
Unfortunately no, `import` is a reserved keyword and cannot be used in ordinary `import` statements.

Reply:
ahhh, I see. Do we still need this file (or is this just here for the tool docs issue?)

Reply:
It's here for the docs generation, and because SC looks here for `pre_process()` / `post_process()`. It's okay if those methods aren't found, but it fails if it can't find the module at all.

Like we talked about today, I'll make a follow-up PR to migrate `make_doc`, `pre_process`, and `post_process` into the task scripts, but I don't want to hold up other changes that rely on this split.